src/plugins/ini/ini.c

```diff
@@ -268,7 +268,7 @@ static void insertKeyIntoKeySet (Key * parentKey, Key * key, KeySet * ks)
 	cursor_t savedCursor = ksGetCursor (ks);
 
 	// trying to find the keys section or sections parent section
-	const char * parent = findParent (parentKey, key, ksDup (ks));
+	char * parent = (char *)findParent (parentKey, key, ksDup (ks));
```

Comment:
findParent was previously returning the keyName of a deleted key, which, although it worked was bad code. but i should probably change the declaration of findParent instead of casting it to avoid a warning.